### ISSUE TITLE
Capistrano 3 Support?

### ISSUE THREAD

**Issue by supairish** [sprinkle-tool/sprinkle#200]
Is it possible to use Sprinkle with Capistrano 3?

**Comment by yyyc514** [comment]
No, but Sprinkle only uses Capistrano internally.  It should be possible to use it side by side with Cap 3 for deployment, might just require some small tweaking.
